packages/core/src/state/reducer.ts

```diff
@@ -69,6 +69,12 @@ export default function reducer(
           [action.meta.key]: result,
         };
         results = applyUpdatersToResults(results, result, action.meta.updaters);
+        if (action.meta.update) {
+          const updaters = action.meta.update(result);
+          Object.keys(updaters).forEach(
+            key => (results[key] = updaters[key](results[key])),
```

Comment:
nit:
```suggestion
            key => { results[key] = updaters[key](results[key]); },
```
seems kind of weird to use the parens rather than a block body